Applications/CMC/cmc.cpp

```diff
@@ -71,6 +72,11 @@ int main(int argc,char **argv)
             return(0);
  
         // PRINT A DEFAULT SETUP FILE FOR THIS INVESTIGATION
+        } else if(startsWith("-j",option.c_str())){
+            char* optionCStr = const_cast<char*>(option.c_str());
+            optionCStr += 2; //increment the char*'s memory address by 2, skip "-j"
+            specifiedMaxNumThreads = atoi(optionCStr);
+
```

Comment:
I like the command line option. As a future step it would be nice to consolidate the handling of command line options for all command line apps.